landingzones/landingzone_secure_vnet_dmz/main.tf

```diff
@@ -0,0 +1,26 @@
+provider "azurerm" {
+  version = "=2.0"
+  features {}
+}
+
+terraform {
+    backend "azurerm" {
+    }
+}
+
+data "terraform_remote_state" "landingzone_caf_foundations" {
+  backend = "azurerm"
+  config = {
+    storage_account_name  = var.lowerlevel_storage_account_name
+    container_name        = var.workspace
+    key                   = "landingzone_caf_foundations.tfstate"
```

Comment:
Do we have to hardcode the tfstate name?

Reply:
Yes we do - by convention/